### PR TITLE
Add thrift middlewares

### PR DESCRIPTION
**Pull request by jlevesy**
@Amoki @AlexisMontagne @phndiaye  RTM ?

### CODE REVIEW

File: lib/rbutils/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
-module Rbutils
-  VERSION = "0.1.0"
+module RbUtils
+  VERSION = '0.1.1'
```
Comment (AlexisMontagne) [inline lib/rbutils/version.rb]: why not `0.0.1`?

File: rbutils.gemspec [hunk 2]
```diff
@@ -5,7 +5,7 @@ require 'rbutils/version'
 
 Gem::Specification.new do |spec|
   spec.name          = 'rbutils'
```
Comment (AlexisMontagne) [inline rbutils.gemspec]: we should call the gem `upfluence-utils` (but keep the repo named `rbutils`)

Comment (jlevesy) [inline rbutils.gemspec]: we should have discussed this before -_-

File: rbutils.gemspec [hunk 3]
```diff
@@ -5,7 +5,7 @@ require 'rbutils/version'
...
-  spec.version       = Rbutils::VERSION
+  spec.version       = RbUtils::VERSION
```
Comment (AlexisMontagne) [inline rbutils.gemspec]: we should namespace all the libs by `Upfluence::Utils`

Comment (jlevesy) [inline rbutils.gemspec]: we should definitely have discussed this before -_-

File: rbutils.gemspec (hunk 2 above)
Comment (AlexisMontagne) [inline rbutils.gemspec]: if you want distribute publicly upfluence specific vendors, we should include the Upfluence name in the gem name

Comment (jlevesy) [inline rbutils.gemspec]: I do agree with that

**Comment by jlevesy** [comment]
@AlexisMontagne  ?

**Comment by AlexisMontagne** [comment]
LGTM

**Pull request merged**